monitor/sth_fetcher_test.go

```diff
@@ -38,6 +38,13 @@ func TestObserveSTH(t *testing.T) {
 	// Make an STH observation
 	m.fetcher.observeSTH()
 
+	// STH fetches are counted
+	fetchObservations := test.CountCounterVecWithLabels(m.fetcher.stats.sthFetchTotal, labels)
+	if fetchObservations != 1 {
```

Comment:
nit: You can combine this line and the previous to reduce the scope of `fetchObservations`:
```suggestion
	if fetchObservations := test.CountCounterVecWithLabels(m.fetcher.stats.sthFetchTotal, labels); fetchObservations != 1 {
```